### PR TITLE
Add several GitHub handles

### PR DESCRIPTION
**Pull request by cthoyt**
As a follow-up to #1655, this PR does the following in a more semi-automated way:

- adds the GitHub handles for @Clare72, @hoganwr, @mmc46, and several others. If anyone wants to help on this scavenger hunt, help would be appreciated!
- updates the contact information in ExO based on the request of @cjgrondin, who is the new maintainer.

## Curation Table/Script

<details>
<summary>Here's the table of all ontologies missing GitHub handles generated from the script below. I removed the rows corresponding to the curation I did in this PR. (click the triangle on the left of this text)</summary>

| contact.name               | contact.email                        | prefix                                                          | repository                                                                     |
|----------------------------|--------------------------------------|-----------------------------------------------------------------|--------------------------------------------------------------------------------|
| Alayne Cuzick              | alayne.cuzick@rothamsted.ac.uk       | [`phipo`](http://www.obofoundry.org/ontology/phipo)             | https://github.com/PHI-base/phipo                                              |
| Alejandra Gonzalez-Beltran | alejandra.gonzalez.beltran@gmail.com | [`stato`](http://www.obofoundry.org/ontology/stato)             | https://github.com/ISA-tools/stato                                             |
| Amanda Hicks               | aellenhicks@gmail.com                | [`htn`](http://www.obofoundry.org/ontology/htn)                 | https://github.com/aellenhicks/htn_owl                                         |
| Antje Chang                | a.chang@tu-bs.de                     | [`bto`](http://www.obofoundry.org/ontology/bto)                 | https://github.com/BRENDA-Enzymes/BTO                                          |
| Bjoern Peters              | bpeters@lji.org                      | [`obi`](http://www.obofoundry.org/ontology/obi)                 | https://github.com/obi-ontology/obi                                            |
| Bob Thacker                | robert.thacker@stonybrook.edu        | [`poro`](http://www.obofoundry.org/ontology/poro)               | https://github.com/obophenotype/porifera-ontology                              |
| Brian Aevermann            | baeverma@jcvi.org                    | [`ogms`](http://www.obofoundry.org/ontology/ogms)               | https://github.com/OGMS/ogms                                                   |
| Carissa Park               | caripark@iastate.edu                 | [`vt`](http://www.obofoundry.org/ontology/vt)                   | https://github.com/AnimalGenome/vertebrate-trait-ontology                      |
| Carrine Blank              | carrine.blank@umontana.edu           | [`micro`](http://www.obofoundry.org/ontology/micro)             | https://github.com/carrineblank/MicrO                                          |
| Ceri Van Slyke             | van_slyke@zfin.org                   | [`zfs`](http://www.obofoundry.org/ontology/zfs)                 | https://github.com/cerivs/zebrafish-anatomical-ontology                        |
| Chris Grove                | cgrove@caltech.edu                   | [`wbls`](http://www.obofoundry.org/ontology/wbls)               | https://github.com/obophenotype/c-elegans-development-ontology                 |
| Chris Grove                | cgrove@caltech.edu                   | [`wbphenotype`](http://www.obofoundry.org/ontology/wbphenotype) | https://github.com/obophenotype/c-elegans-phenotype-ontology                   |
| Citlalli Mejía-Almonte     | citlalli.mejiaalmonte@gmail.com      | [`mco`](http://www.obofoundry.org/ontology/mco)                 | https://github.com/microbial-conditions-ontology/microbial-conditions-ontology |
| Damion Dooley              | damion_dooley@sfu.ca                 | [`genepio`](http://www.obofoundry.org/ontology/genepio)         | https://github.com/GenEpiO/genepio                                             |
| Danielle Welter            | danielle.welter@uni.lu               | [`hancestro`](http://www.obofoundry.org/ontology/hancestro)     | https://github.com/EBISPOT/ancestro                                            |
| David Orloff               | dorloff@ncmir.ucsd.edu               | [`fbbi`](http://www.obofoundry.org/ontology/fbbi)               | https://github.com/CRBS/Biological_Imaging_Methods_Ontology                    |
| Edison Ong                 | edong@umich.edu                      | [`ohpi`](http://www.obofoundry.org/ontology/ohpi)               | https://github.com/OHPI/ohpi                                                   |
| Fernanda Farinelli         | fernanda.farinelli@gmail.com         | [`ontoneo`](http://www.obofoundry.org/ontology/ontoneo)         | https://github.com/ontoneo-project/Ontoneo                                     |
| G. Thomas Hayman           | gthayman@mcw.edu                     | [`pw`](http://www.obofoundry.org/ontology/pw)                   | https://github.com/rat-genome-database/PW-Pathway-Ontology                     |
| George Gkoutos             | g.gkoutos@gmail.com                  | [`uo`](http://www.obofoundry.org/ontology/uo)                   | https://github.com/bio-ontology-research-group/unit-ontology                   |
| Gerhard Mayer              | gerhard.mayer@rub.de                 | [`ms`](http://www.obofoundry.org/ontology/ms)                   | https://github.com/HUPO-PSI/psi-ms-CV                                          |
| Gerhard Mayer              | gerhard.mayer@rub.de                 | [`xlmod`](http://www.obofoundry.org/ontology/xlmod)             | https://github.com/HUPO-PSI/mzIdentML                                          |
| Hector Escriva             | hescriva@obs-banyuls.fr              | [`amphx`](http://www.obofoundry.org/ontology/amphx)             | https://github.com/EBISPOT/amphx_ontology                                      |
| Huang, Jingshan            | huang@southalabama.edu               | [`omit`](http://www.obofoundry.org/ontology/omit)               | https://github.com/OmniSearch/omit                                             |
| JAX phenotype list         | pheno@jax.org                        | [`mp`](http://www.obofoundry.org/ontology/mp)                   | https://github.com/obophenotype/mammalian-phenotype-ontology                   |
| James C. Hu                | jimhu@tamu.edu                       | [`omp`](http://www.obofoundry.org/ontology/omp)                 | https://github.com/microbialphenotypes/OMP-ontology                            |
| Janna Hastings             | janna.hastings@gmail.com             | [`mfomd`](http://www.obofoundry.org/ontology/mfomd)             | https://github.com/jannahastings/mental-functioning-ontology                   |
| Jennifer C. Girón          | entiminae@gmail.com                  | [`aism`](http://www.obofoundry.org/ontology/aism)               | https://github.com/insect-morphology/aism                                      |
| Jennifer Smith             | jrsmith@mcw.edu                      | [`mmo`](http://www.obofoundry.org/ontology/mmo)                 | https://github.com/rat-genome-database/MMO-Measurement-Method-Ontology         |
| Jennifer Smith             | jrsmith@mcw.edu                      | [`xco`](http://www.obofoundry.org/ontology/xco)                 | https://github.com/rat-genome-database/XCO-experimental-condition-ontology     |
| Jie Zheng                  | jiezheng@pennmedicine.upenn.edu      | [`obcs`](http://www.obofoundry.org/ontology/obcs)               | https://github.com/obcs/obcs                                                   |
| Jie Zheng                  | jiezheng@pennmedicine.upenn.edu      | [`obib`](http://www.obofoundry.org/ontology/obib)               | https://github.com/biobanking/biobanking                                       |
| Jonathan Bard              | J.Bard@ed.ac.uk                      | [`aeo`](http://www.obofoundry.org/ontology/aeo)                 | https://github.com/obophenotype/human-developmental-anatomy-ontology           |
| Karen Eilbeck              | keilbeck@genetics.utah.edu           | [`so`](http://www.obofoundry.org/ontology/so)                   | https://github.com/The-Sequence-Ontology/SO-Ontologies                         |
| Laurel Cooper              | cooperl@science.oregonstate.edu      | [`pso`](http://www.obofoundry.org/ontology/pso)                 | https://github.com/Planteome/plant-stress-ontology                             |
| Lynn Schriml               | lynn.schriml@gmail.com               | [`symp`](http://www.obofoundry.org/ontology/symp)               | https://github.com/DiseaseOntology/SymptomOntology                             |
| Lynn Schriml               | lynn.schriml@gmail.com               | [`trans`](http://www.obofoundry.org/ontology/trans)             | https://github.com/DiseaseOntology/PathogenTransmissionOntology                |
| Martin Ramirez             | ramirez@macn.gov.ar                  | [`spd`](http://www.obofoundry.org/ontology/spd)                 | https://github.com/obophenotype/spider-ontology                                |
| Mary Shimoyama             | shimoyama@mcw.edu                    | [`cmo`](http://www.obofoundry.org/ontology/cmo)                 | https://github.com/rat-genome-database/CMO-Clinical-Measurement-Ontology       |
| Mathias Brochhausen        | MBrochhausen@uams.edu                | [`apollo_sv`](http://www.obofoundry.org/ontology/apollo_sv)     | https://github.com/ApolloDev/apollo-sv                                         |
| Mathias Brochhausen        | mbrochhausen@gmail.com               | [`oostt`](http://www.obofoundry.org/ontology/oostt)             | https://github.com/OOSTT/OOSTT                                                 |
| Matthew Brush              | mhb120@gmail.com                     | [`sepio`](http://www.obofoundry.org/ontology/sepio)             | https://github.com/monarch-initiative/SEPIO-ontology                           |
| Meng LIU                   | liumeng94@sjtu.edu.cn                | [`iceo`](http://www.obofoundry.org/ontology/iceo)               | https://github.com/ontoice/ICEO                                                |
| Nicolas Le Novere          | n.lenovere@gmail.com                 | [`mamo`](http://www.obofoundry.org/ontology/mamo)               |                                                                                |
| Onard Mejino               | mejino@u.washington.edu              | [`fma`](http://www.obofoundry.org/ontology/fma)                 |                                                                                |
| Pablo Porras Millán        | pporras@ebi.ac.uk                    | [`mi`](http://www.obofoundry.org/ontology/mi)                   | https://github.com/HUPO-PSI/psi-mi-CV                                          |
| Pankaj Jaiswal             | jaiswalp@science.oregonstate.edu     | [`peco`](http://www.obofoundry.org/ontology/peco)               | https://github.com/Planteome/plant-experimental-conditions-ontology            |
| Pankaj Jaiswal             | jaiswalp@science.oregonstate.edu     | [`to`](http://www.obofoundry.org/ontology/to)                   | https://github.com/Planteome/plant-trait-ontology                              |
| Paul Fabry                 | paul.fabry@usherbrooke.ca            | [`cvdo`](http://www.obofoundry.org/ontology/cvdo)               | https://github.com/OpenLHS/CVDO                                                |
| Paul Fabry                 | paul.fabry@usherbrooke.ca            | [`labo`](http://www.obofoundry.org/ontology/labo)               | https://github.com/OpenLHS/LABO                                                |
| Paul Schofield             | pns12@hermes.cam.ac.uk               | [`mpath`](http://www.obofoundry.org/ontology/mpath)             | https://github.com/PaulNSchofield/mpath                                        |
| Pierre-Alain Binz          | pierre-alain.binz@chuv.ch            | [`mod`](http://www.obofoundry.org/ontology/mod)                 | https://github.com/HUPO-PSI/psi-mod-CV                                         |
| Priti Parikh               | preets79@gmail.com                   | [`opl`](http://www.obofoundry.org/ontology/opl)                 | https://github.com/OPL-ontology/OPL                                            |
| Rahuman Sheriff            | sheriff@ebi.ac.uk                    | [`sbo`](http://www.obofoundry.org/ontology/sbo)                 |                                                                                |
| Raymond Lee                | raymond@caltech.edu                  | [`wbbt`](http://www.obofoundry.org/ontology/wbbt)               | https://github.com/obophenotype/c-elegans-gross-anatomy-ontology               |
| SED-ML Editors             | sed-ml-editors@googlegroups.com      | [`kisao`](http://www.obofoundry.org/ontology/kisao)             | https://github.com/SED-ML/KiSAO                                                |
| Shur-Jen Wang              | sjwang@mcw.edu                       | [`rs`](http://www.obofoundry.org/ontology/rs)                   | https://github.com/rat-genome-database/RS-Rat-Strain-Ontology                  |
| Sirarat Sarntivijai        | siiraa@umich.edu                     | [`clo`](http://www.obofoundry.org/ontology/clo)                 | https://github.com/CLO-Ontology/CLO                                            |
| Stephen Fisher             | safisher@upenn.edu                   | [`ornaseq`](http://www.obofoundry.org/ontology/ornaseq)         | https://github.com/safisher/ornaseq                                            |
| Suzi Aleksander            | suzia@stanford.edu                   | [`go`](http://www.obofoundry.org/ontology/go)                   | https://github.com/geneontology/go-ontology                                    |

</details>

Here's the Python code used to generate this table:

```python
from bioregistry.external.obofoundry import get_obofoundry
from tabulate import tabulate


def main():
    rows = []
    headers = ["contact.name", "contact.email", "prefix", "repository"]
    for prefix, data in get_obofoundry().items():
        if data.get("inactive") or data.get("deprecated"):
            continue
        github = data.get("contact.github")
        if github:
            continue
        rows.append((
            data.get("contact.label"),
            data.get("contact"),
            f"[`{prefix}`](http://www.obofoundry.org/ontology/{prefix})",
            data.get("repository"),
        ))
    rows = sorted(rows)
    print(tabulate(rows, headers=headers, tablefmt="github"))


if __name__ == '__main__':
    main()
```

### CODE REVIEW

**Comment by cthoyt** [comment]
Two that I will address in alternate PRs: `mp` has a listserve as its contact information that should probably be @sbello, @yongqunh has an inconsistent use of his name (not sure if that's on purpose or not), and a few more

**Comment by mmc46** [comment]
I'm no longer in Flybase or involved in maintaining the ontologies. I'm not sure if `fbsp` should have @Clare72 as the contact.

**Comment by Clare72** [comment]
I don't think anybody is maintaining it any more and I have never worked on it at all, so maybe it is not appropriate for there to be a contact? I guess it could be me, but I don't really know anything about it if anyone does actually want more information!

**Comment by matentzn** [comment]
Is it necessary to add contact information to obsolete ontologies though? Seems a bit overkill.. Inactive, ok, but obsolete..

**Comment by matentzn** [comment]
Knock yourself out, but remember that the metadata records of the various ontologies should not be changed for non-technical reasons without the authors consent.. Make sure you get the permission.. Good effort! I love metadata being improved! :)

**Comment by cthoyt** [comment]
I would 100% consider adding GitHub handles as a technical fix of metadata; I feel confident in not needing consent to add information that's possible to acquire by using the publicly available search feature on GitHub.

That only leaves the update requested from Cynthia. I'll revert the place where I normalized Bill Hogan's name.

**Comment by sbello** [comment]
The mp contact is actually an email alias that includes me, Anna, and Cindy which makes it useful as that way we all get the contact and can respond even if one or two of us are away. Not sure why it shows up as a mailing list. I would prefer to keep this. If we want to add github handles as well that is fine.

**Comment by matentzn** [comment]
I am converting this to draft until it is ready to be merged. What are the action items?

**Comment by matentzn** [comment]
Actually, I looked this over and apart from @zachll, I know everyone personally (digitally :P) and I am confident they would not object to the addition of their github handles. @zachll. Hope this is fine for the ontologies assigned you if not, please make a ticket and we will remove it.

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
@Clare72 I assigned you as contact to fbsp, but I don't really expect anyone to contact you. If anyone should, just say its out of commission, and you can ask David for the history if needed.

**Comment by matentzn** [comment]
Thank you 1000x @cthoyt for that!
